File: Array/LargestElement.cpp
#include<bits/stdc++.h>
using namespace std;

// function to find maximum
int largest(int arr[], int n){
    int max = arr[0];
    for(int i=1; i<n; i++){ 
        if(arr[i] > max){
            max = arr[i];
        }
    }
    return max;
}

// driver code 
int main(){
    int arr[] = {10,12,5,44,70};
    int n = sizeof(arr)/sizeof(arr[0]);
    cout<< largest(arr,n);
    return 0;
}

File: Array/LargestElement.py
import time

def largest(arr):
    max = arr[0]
    for i in range(1,len(arr)):
        if arr[i] > max:
            max = arr[i]
    
    return max

start_time = time.time()
arr = [12,23,44,777,1]
result = largest(arr)
end_time = time.time()
execution_time = end_time - start_time

print(result)
print(f"Execution time: {execution_time:.6f} seconds")
